packages/nestjs/src/index.ts

```diff
@@ -1,8 +1,16 @@
 import type { Express, Request as ExpressRequest, Response as ExpressResponse } from 'express';
 import type { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
-import { printSchema } from 'graphql';
-import { createYoga, filter, pipe, YogaServerInstance, YogaServerOptions } from 'graphql-yoga';
+import { GraphQLSchema, printSchema } from 'graphql';
+import {
+  createYoga,
+  filter,
+  pipe,
+  YogaSchemaDefinition,
+  YogaServerInstance,
+  YogaServerOptions,
+} from 'graphql-yoga';
 import type { ExecutionParams } from 'subscriptions-transport-ws';
+import { mergeSchemas } from '@graphql-tools/schema';
```

Comment:
Since we're changing graphql-yoga itself, it would make sense to re-export the `mergeSchema` from it too (since graphql-yoga has @graphql-tools/schema as a dependency) instead of adding the dependency to the nestjs plugin.

```suggestion
  mergeSchema,
} from 'graphql-yoga';
import type { ExecutionParams } from 'subscriptions-transport-ws';
```

Reply:
Yes you're right, it will be better to do it that way, I'm fixing it !

Reply:
Done